doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -9,23 +9,24 @@ sort_idx: 20
 * TOC
 {:toc}
 
-The *Common Type Library (CTL)* is a repository of reusable data type schemas that can be referenced and reused as the user creates specific schema definitions for Kaa modules.
-The *Common Type (CT)* is a unit of CTL which represents set of *data type schema versions (CT schemas)*.
-This capability unifies management of all schemas in a Kaa server instance.
+The [Common type library (CTL)]({{root_url}}Glossary/#common-type-library-ctl) is a repository of data type schemas used for [endpoint profiling]({{root_url}}Glossary/#endpoint-profile).
```

Comment:
Not only endpoint profiles. Starting from Kaa 0.10.0 CT was integrated for the rest of Kaa modules.